iidm/iidm-extensions/src/test/java/com/powsybl/iidm/network/extensions/MeasurementTest.java

```diff
@@ -0,0 +1,119 @@
+package com.powsybl.iidm.network.extensions;
```

Comment:
The copyright should be added:

```suggestion
/**
 * Copyright (c) 2025, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 * SPDX-License-Identifier: MPL-2.0
 */
package com.powsybl.iidm.network.extensions;
```

Reply:
Done